docs/proposals/202101_endpoint_discovery.md

```diff
@@ -19,17 +21,17 @@ Currently, in Thanos Query, the discovery of rules APIs happens via Store API's
 
 We also require passing two different flags to the Thanos Query component `--store=<address>` and `--rule=<address>`. If users use both flags, the Query component performs DNS discovery on often the same address multiple times, this can cause occasional DNS issues as this resolving happens so frequently. This is especially confusing when one works and the other doesn't.
 
-Adding new APIs in the future, like
+Adding new APIs in the future alike.
```

Comment:
This sound a bit weird. `alike` adverb means:

```
in the same or a similar way.
```

So it is missing extra information. Perhaps:

```suggestion
Adding new APIs in the future would exacerbate this issue.
```

Reply:
I think we should add this in the previous paragraph because it is also talking about the same thing?